text/src/autogluon/text/automm/models/clip.py

```diff
@@ -98,28 +118,60 @@ def forward(
 
         assert images.dim() == 5
         b, n, c, h, w = images.shape
-        image_features = self.model.get_image_features(
+        vision_outputs = self.model.vision_model(
```

Comment:
Just a question, why we want to do this naming change, like from image -> vision?

Reply:
Changing from `get_image_features` to `vision_model` is to be consistent with changing from `get_text_features` to `text_model`. See https://github.com/huggingface/transformers/blob/v4.14.1/src/transformers/models/clip/modeling_clip.py#L1006-L1010

Reply:
I see, thank you for the reference.